### PR TITLE
Fix #14094 - Improve strings for cache list info items

### LINKED ISSUE #14094: Improve strings for list info item configuration
### Describe your problem!

To be platform neutral we should not use "GC Code" but "geo code" as we do in cache details.

Sorry for only writing an issue, but dev environment is not with me to fix it instantly.

Maybe we can (in general?) also prefer to reuse existing strings for such new features to make it easier to keep consistent wording and avoid additional translation effort at the same time. I guess most of the items of the list items config is already existing somewhere (e.g. in cache details view).

### How to reproduce?

Check string cacheListInfoItem_GCCode

### Actual result after these steps?

_No response_

### Expected result after these steps?

_No response_

### Reproducible

Yes

### c:geo Version

2023.03.12-NB

### System information

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Lineflyer**
## Description

Proposal for improved strings for this new feature

## Related issues

#14094

### CODE REVIEW

File: main/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -959,23 +959,23 @@
...
+    <string name="init_title_cacheListInfo2">Configure list info lines (Line 2)</string>
+    <string name="init_summary_cacheListInfo">Select the additional information to be shown for each cache in cache lists</string>
     <string name="init_livelist">Show Direction</string>
     <string name="init_summary_livelist">Show an arrow with compass direction for caches in a list</string>
     <string name="init_quicklaunchitems">Quick launch buttons</string>
     <string name="init_summary_quicklaunchitems">Select one or more items as quick launch buttons on home screen</string>
 
     <!-- names for CacheListInfoItem -->
-    <string name="cacheListInfoItem_GCCode">GC Code</string>
+    <string name="cacheListInfoItem_GCCode">Geocode</string>
```
Comment (bekuno) [inline main/src/main/res/values/strings.xml:971]: I propose to also change the string name here.

Comment (Lineflyer) [reply main/src/main/res/values/strings.xml:971]: mmh..strange. What am I doing wrong. I used the context option "Find usage":
image

Comment (Lineflyer) [reply main/src/main/res/values/strings.xml:971]: found it. Commit incoming

**Comment by moving-bits** [comment]
FYI: To rename a variable (string or other) in AS, you can mark the variable name, press Shift-F6, and enter the new name. AS will then automatically rename all occurrences of the var (taking variable scope into account).

**Comment by bekuno** [comment]
Should this also work for enum values?

**Review by moving-bits — APPROVED** [review]

**Pull request merged**